smokey/src/Components/Ingredients.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Ingredients(singleIngredient) {
```

Comment:
Name is Ingredients but looks like it just renders one ingredient. 

Also are we concerned with nil checks?
```
singleIngredient.singleIngredient?.ingredient_name
```

Why is singleIngredient nested in another singleIngredient?